### ISSUE TITLE
ugs will not install on my MAC 14.2.1 laptop

### ISSUE THREAD

**Issue by StephenPateMorales** [davidchall/homebrew-hep#382]
### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I am trying to install ugs so that I can install topdrawer.  But the ugs installation fails.

### What happened (include all command output)?

pate@SFP-MacBook-Pro-Work ~ % brew install ugs
==> Downloading https://formulae.brew.sh/api/formula.jws.json

==> Downloading https://formulae.brew.sh/api/cask.jws.json

==> Fetching davidchall/hep/ugs
==> Downloading https://gist.githubusercontent.com/andriish/4df18144d5992c0d3301e3f0069b9218/raw/e18c51f5657b9ec8c519ed25cba838fc353fc35d/
Already downloaded: /Users/pate/Library/Caches/Homebrew/downloads/5701a3bd90eb3ac5a3abe8e506e3aef89112868b00c5fa13fbfcebb9ff24b03f--patch-ugs-1.txt
==> Downloading https://ftp.riken.jp/iris/ugs/ugs.tar.gz
Already downloaded: /Users/pate/Library/Caches/Homebrew/downloads/b251d4d8162276f9f5e2ed7ec205dfb3e379ed6a5b017f939c3b18e09dcf0c97--ugs.tar.gz
==> Installing ugs from davidchall/hep
==> Patching
==> Applying patch-ugs-1.txt
patching file Imakefile.def
patching file 'src.2.10e/Imakefile'
patching file 'src.2.10e/Makefile.tmpl'
patching file 'src.2.10e/ugclos.f'
patching file 'src.2.10e/ugdupl.f'
patching file 'src.2.10e/ugopen.F'
patching file 'src.2.10e/ugslct.f'
==> xmkmf
==> make Makefiles
==> make clean
==> make
Last 15 lines from /Users/pate/Library/Logs/Homebrew/ugs/04.make:
      _MAIN__ in ccuym0IZ.o
      _MAIN__ in ccuym0IZ.o
      ...
  _ugopen_, referenced from:
      _MAIN__ in ccuym0IZ.o
  _ugplin_, referenced from:
      _MAIN__ in ccuym0IZ.o
      _MAIN__ in ccuym0IZ.o
  _ugslct_, referenced from:
      _MAIN__ in ccuym0IZ.o
  _ugwrit_, referenced from:
      _MAIN__ in ccuym0IZ.o
collect2: error: ld returned 1 exit status
make[1]: *** [tryxw] Error 1
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/davidchall/homebrew-hep/issues

pate@SFP-MacBook-Pro-Work ~ % 

### What did you expect to happen?

I expected to install ugs.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install ugs
```

### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
pate@SFP-MacBook-Pro-Work ~ % brew config
HOMEBREW_VERSION: 4.2.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: e5fefd73cd97cd36ae3af29551f529ae59b333d6
Last commit: 3 days ago
Core tap JSON: 22 Feb 01:29 UTC
Core cask tap JSON: 22 Feb 01:29 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.93sZKRnT6Y/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.2.1-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: 15.2
Rosetta 2: false
pate@SFP-MacBook-Pro-Work ~ % brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1
pate@SFP-MacBook-Pro-Work ~ %
```

**Comment by davidchall** [comment]
Hi @StephenPateMorales - thanks for the report. This should be resolved by #384 - can you try again?

**Comment by StephenPateMorales** [comment]
Thank you, that worked!
